src/client/Item/ItemDefinition.ts

```diff
@@ -7,5 +7,6 @@
  * in {@link ItemBody}
  */
 export interface ItemDefinition {
```

Comment:
non-blocking comment: should we even have this object? It's almost useless.

Reply:
I don't like it either